### ISSUE TITLE
PSR-4 Warning during composer installation

### ISSUE THREAD

**Issue by fazzinipierluigi** [microsoftgraph/msgraph-sdk-php#791]
Hello!
I'm tryng to install this sdk with composer on my brand new Laravel 9 installation.
But during installation i'm otain these bad warnings:
```
Class Beta\Microsoft\Graph\Security\Model\Alert located in ###/vendor/microsoft/microsoft-graph/src/Beta/Microsoft/Graph\SecurityNamespace\Model\Alert.php does not comply with psr-4 autoloading standard. Skipping.                                                                                                                                              
Class Beta\Microsoft\Graph\Security\Model\AlertClassification located in ###/vendor/microsoft/microsoft-graph/src/Beta/Microsoft/Graph\SecurityNamespace\Model\AlertClassification.php does not comply with psr-4 autoloading standard. Skipping.                                                                                                                  
Class Beta\Microsoft\Graph\Security\Model\AlertComment located in ###/vendor/microsoft/microsoft-graph/src/Beta/Microsoft/Graph\SecurityNamespace\Model\AlertComment.php does not comply with psr-4 autoloading standard. Skipping.                                                                                                                                
Class Beta\Microsoft\Graph\Security\Model\AlertDetermination located in ###/vendor/microsoft/microsoft-graph/src/Beta/Microsoft/Graph\SecurityNamespace\Model\AlertDetermination.php does not comply with psr-4 autoloading standard. Skipping.                                                                                                                    
Class Beta\Microsoft\Graph\Security\Model\AlertSeverity located in ###/vendor/microsoft/microsoft-graph/src/Beta/Microsoft/Graph\SecurityNamespace\Model\AlertSeverity.php does not comply with psr-4 autoloading standard. Skipping.                                                                                                                              
Class Beta\Microsoft\Graph\Security\Model\AlertStatus located in ###/vendor/microsoft/microsoft-graph/src/Beta/Microsoft/Graph\SecurityNamespace\Model\AlertStatus.php does not comply with psr-4 autoloading standard. Skipping.                                                                                                                                  
Class Beta\Microsoft\Graph\Security\Model\DetectionSource located in ###/vendor/microsoft/microsoft-graph/src/Beta/Microsoft/Graph\SecurityNamespace\Model\DetectionSource.php does not comply with psr-4 autoloading standard. Skipping.                                                                                                                          
Class Beta\Microsoft\Graph\Security\Model\HuntingQueryResults located in ###/vendor/microsoft/microsoft-graph/src/Beta/Microsoft/Graph\SecurityNamespace\Model\HuntingQueryResults.php does not comply with psr-4 autoloading standard. Skipping.                                                                                                                  
Class Beta\Microsoft\Graph\Security\Model\HuntingRowResult located in ###/vendor/microsoft/microsoft-graph/src/Beta/Microsoft/Graph\SecurityNamespace\Model\HuntingRowResult.php does not comply with psr-4 autoloading standard. Skipping.                                                                                                                        
Class Beta\Microsoft\Graph\Security\Model\Incident located in ###/vendor/microsoft/microsoft-graph/src/Beta/Microsoft/Graph\SecurityNamespace\Model\Incident.php does not comply with psr-4 autoloading standard. Skipping.                                                                                                                                        
Class Beta\Microsoft\Graph\Security\Model\IncidentStatus located in ###/vendor/microsoft/microsoft-graph/src/Beta/Microsoft/Graph\SecurityNamespace\Model\IncidentStatus.php does not comply with psr-4 autoloading standard. Skipping.                                                                                                                            
Class Beta\Microsoft\Graph\Security\Model\ServiceSource located in ###/vendor/microsoft/microsoft-graph/src/Beta/Microsoft/Graph\SecurityNamespace\Model\ServiceSource.php does not comply with psr-4 autoloading standard. Skipping.                                                                                                                              
Class Beta\Microsoft\Graph\Security\Model\SinglePropertySchema located in ###/vendor/microsoft/microsoft-graph/src/Beta/Microsoft/Graph\SecurityNamespace\Model\SinglePropertySchema.php does not comply with psr-4 autoloading standard. Skipping. 
```

**Comment by Orrison** [comment]
I see this is in triage now. Any plans to get this bug fixed?

**Comment by AndMe2** [comment]
Getting same, all under directory "/vendor/microsoft/microsoft-graph/src/Beta/Microsoft/Graph/SecurityNamespace/Model/"

Should rename either directory from "SecurityNamespace" to "Security", or all namespace in files to "SecurityNamespace"

I think top level must be named same to comply to psr-4.

Class Beta\Microsoft\Graph\Security\Model\JustifyAction located in ./vendor/microsoft/microsoft-graph/src/Beta/Microsoft/Graph/SecurityNamespace/Model/JustifyAction.php does not comply with psr-4 autoloading standard. Skipping.
and so on...

**Comment by Elshaden** [comment]
@Ndiritu 
Any Update on this issue.. 

Thanks

**Comment by Ndiritu** [comment]
Sorry for the delay in resolving this. The latest release `1.64.0` contains the bug-fix

**Comment by hebbet** [comment]
that problem is back with the latest Version

**Comment by Ndiritu** [comment]
(Note to self)
Re-introduced by [IdentityGovernanceNamespace](https://github.com/microsoftgraph/msgraph-sdk-php/pull/975/files) models

**Comment by Kyle-Jeynes** [comment]
On `^1.74` this is still happening. What can I do to resolve this in the meantime? Downgrade to the last (by the looks of it) stable build of `1.0.64`?

**Comment by Ndiritu** [comment]
Apologies for the delay. This has been resolved in `1.75.0`
